src/test/java/ShareLaneTest.java

```diff
@@ -0,0 +1,159 @@
+import net.datafaker.Faker;
+import org.openqa.selenium.By;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class ShareLaneTest {
+
+    Faker faker = new Faker();
+
+    @Test
+    public void zipCodePositiveTest() {
+
+        RemoteWebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+
+
+        driver.navigate().to("https://www.sharelane.com/cgi-bin/register.py");
+
+        driver.findElement(By.name("zip_code")).sendKeys("12345");
+        driver.findElement(By.cssSelector("input[value=Continue]")).click();
+        boolean findInputRegister = driver.findElement(By.cssSelector("input[value=Register]")).isDisplayed();
+
+        Assert.assertTrue(findInputRegister);
+
+        driver.quit();
+    }
+
+
+    @Test
+    public void zipCodeIsEmpty() {
+
+        RemoteWebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+
+
+        driver.navigate().to("https://www.sharelane.com/cgi-bin/register.py");
+
+        driver.findElement(By.name("zip_code")).sendKeys("");
+        driver.findElement(By.cssSelector("input[value=Continue]")).click();
+        boolean findErrorMessage = driver.findElement(By.className("error_message")).isDisplayed();
+
+        Assert.assertTrue(findErrorMessage);
+
+        driver.quit();
+    }
+
+    @Test
+    public void zipCodeEnter4digit() {
```

Comment:
do  not use numbers in variables / method names, it's disallowed according to the java code convention.
If it's really needed to add number to the name - type it with lettrs e.g. 4 - "Four"